apps/bandwagon/forms.py

```diff
@@ -143,6 +145,12 @@ def __init__(self, *args, **kw):
                 self.instance.type in amo.COLLECTION_SPECIAL_SLUGS):
             del self.fields['slug']
 
+    def clean(self):
```

Comment:
could instead be done in `clean_your_name` right? If you put it in `clean` on purpose (to not have the error message associated with this field maybe? So it's not obvious that it's a honeypot?) then please add a comment here explaining that.